packages/delayedInbox-l2msg/scripts/withdrawFunds.js

```diff
@@ -0,0 +1,168 @@
+const { providers, Wallet, ethers } = require('ethers')
+const { arbLog, requireEnvVariables } = require('arb-shared-dependencies')
+const { getL2Network } = require("@arbitrum/sdk-nitro/dist/lib/dataEntities/networks")
+requireEnvVariables(['DEVNET_PRIVKEY', 'L2RPC', 'L1RPC'])
+
+/**
+ * Set up: instantiate L1 / L2 wallets connected to providers
+ */
+const walletPrivateKey = process.env.DEVNET_PRIVKEY
+
+const L2MSG_signedTx = 4
+
+const l1Provider = new providers.JsonRpcProvider(process.env.L1RPC)
+const l2Provider = new providers.JsonRpcProvider(process.env.L2RPC)
+
+const l1Wallet = new Wallet(walletPrivateKey, l1Provider)
+const l2Wallet = new Wallet(walletPrivateKey, l2Provider)
+
+/**
+   * We should use nodeInterface to get the gas estimate is because we 
+   * are making a delayed inbox message which doesn't need l1 calldata
+   * gas fee part.
+   */
+ const estimateGasWithoutL1Part = async (transactionl2Request) => {
+  const iface = new ethers.utils.Interface(NodeInterface__factory.abi)
+  const calldata = iface.encodeFunctionData('gasEstimateComponents', [transactionl2Request.to, false, transactionl2Request.data])
+  const transactionEstimateRequest = {
+    data: calldata,
+    to: NODE_INTERFACE_ADDRESS,
+    from: l1Wallet.address,
+  }
+  const res = await l2Wallet.call(transactionEstimateRequest)
+  const feeData = iface.decodeFunctionResult('gasEstimateComponents',res)
+  return feeData.gasEstimate.sub(feeData.gasEstimateForL1)
```

Comment:
no need to do this manually
```suggestion
    const gasComponents = (await nodeInterface.callStatic.gasEstimateComponents(transactionl2Request.to, false, transactionl2Request.data, {
      from: transactionl2Request.from
    }))
    return gasComponents.gasEstimate.sub(gasComponents.gasEstimateForL1)
```